app/demo-interactor/simple-driver.py

```diff
@@ -29,14 +29,18 @@
     }
 }
 
+exe = environ.get('CELERITAS_DEMO_EXE', './demo-interactor')
+
 print("Input:")
-pprint(inp)
+with open(f'{exe}.inp.json', 'w') as f:
+    json.dump(inp, f, indent=1)
+print(json.dumps(inp, indent=1))
```

Comment:
I think you can  delete lines 45/46 with this change. Also I think pprint is no longer used -- so you can delete the import.